modules/ROOT/pages/clustering/databases.adoc

```diff
@@ -225,6 +225,124 @@ neo4j@neo4j> DRYRUN REALLOCATE DATABASES;
+
+Remember that the recreate procedure results in downtime while the stores get updated.
+The time is unbounded and may depend on different factors -- for example, the size of the store, network speed, etc.
+====
+
+The database in question can be in an `online` or `offline` state when it is recreated, but a successful operation starts the database regardless of its previous state.
+
+If CDC is enabled for the database, it will still be enabled after recreating the database, however the chain will be broken.
+See link:https://neo4j.com/docs/cdc/current/backup-restore/[CDC documentation -> Restore backups and snapshots] for more details.
+
+Before recreating a database, any eventual quarantined states need to be addressed.
+For more information, see xref:database-administration/standard-databases/errors.adoc#quarantine[Standard databases -> Error handling].
+
+You need xref:authentication-authorization/dbms-administration.adoc#access-control-dbms-administration-database-management[the `CREATE DATABASE` and `DROP DATABASE` privileges] to run the recreate procedure.
+
+To check if the recreation is successful, use the `SHOW DATABASES` command and verify that all allocations have been started.
+
+Additionally, you have the option to modify <<alter-topology-recreate, the topology>> during the recreation process.
+However, note that the store format, access, and enrichment cannot be altered during recreation.
+
+[[recreate-seeding-options]]
+=== Seeding options
+
+The store to be used during the recreation of a database can be defined in different ways.
+One method uses a backup, while others use available allocations in the cluster.
+
+You can use either <<uri-seed, `seedURI`>> or <<seed-servers, `seedingServers`>> to specify the source from which the database should be recreated.
+
+* If you define neither, an error is thrown.
+
+* If you define both of them, then `seedingServers` must be an empty list.
+
+* If `seedingServers` is not empty and `seedURI` is also defined, an error will occur.
+See <<undefined-servers-backup, Undefined servers with fallback backup>> for more details.
+
+[[uri-seed]]
+==== Use backup as a seed
+
+If you provide a URI to a backup or a dump, the stores on all allocations will be replaced by the backup or the dump at the given URI.
+The new allocations can be put on any `ENABLED` server in the cluster.
+See <<cluster-seed-uri, Seed from URI>> for more details.
+
+To avoid possible data loss, ensure you back up the current store before replacing it with a seed from a URI, so the latest state can be restored if needed.
+
+[source, shell]
+----
+CALL dbms.cluster.recreateDatabase("neo4j", {seedURI: "s3:/myBucket/myBackup.backup"});
+----
+
+[[seed-servers]]
+==== Use available allocations as seed
+
+The database recreation process is designed to select the latest seed from the seeding servers.
+Specifying only a subset of the previous cluster servers as seeders carries a risk of data loss.
```

Comment:
What about this suggestions:

After the recreation has finished, the database will have the most up-to-date store that could be found on the seeding servers. 

And after that in a "caution" block:
Recreation is based on remaining stores or store defined by the user. This means that stores which were lost or not defined is not used for the recreation. If not used stores were more up to date than the used ones, this will result in data loss.